test/test_helper.js

```diff
@@ -0,0 +1,4 @@
+process.env.NODE_ENV = 'test'
+require('../app.js')
+require('../lib/globals')
+global.expect = require('chai').expect
```

Comment:
This is great that expect is in one place and everyone else can get it from this helper file. Only issue I guess is standard is going to complain saying expect is not defined.